src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -63,5 +65,25 @@ class JesBackendCall(val backend: JesBackend,
    */
   def globOutputPath(glob: String) = s"$callGcsPath/glob-${glob.md5Sum}/"
 
-  override def resume(jobKey: JobKey) = backend.resume(this, jobKey)
+  /** TODO creating the JES run should be async but currently is not. */
+  override def execute = Future.successful(backend.execute(this))
```

Comment:
Yes I meant these to block when I wrote them, though I'll try to unblock them before merge.  I did this mostly to keep things simple while I worked through the refactoring and also to better understand where the issues were with blocking.  It turns out this blocking code doesn't appear to cause concurrency issues: Cromwell can now go at least 500 workflows/calls wide on JES, while before these changes it wouldn't go past 24.  The real concurrency problem appears to have been (not surprisingly) the explicit `Thread.sleep`s between JES polls.

As I said I would like to unblock these because while this structure appears good enough to expose the next bottleneck, it might not be good enough if we want to scale even wider.